README.md

```diff
@@ -13,25 +13,59 @@ Please visit [Demo AvoRed Rust CMS Admin](https://demo-admin.avored.com)
 
 ## Installation
 
-    git clone https://github.com/avored/avored-rust-cms.git
-    cd avored-cms
-    cp .env .env.dev    //Basically copy the .env.example file to .env
-    
+Make sure the prerequisite are installed and then go through the install steps to install a local build of Avored.
 
-Set up your `.env.dev` file you can rename the existing `.env.prod` file make sure the database folder name, password salt, jwt secret setup properly random string and smtp information setup right if you wanted to use the forgot password feature(optional) NOTE: for local dev smtp info can be ignored.
+### Prerequisites
 
-### Start your application backend
-    cargo run
+**Protobuf**, Google's data interchange format written in C++, is required to be installed before build.  Read more about it at the protocol buffers website: https://protobuf.dev/ or at the repository https://github.com/protocolbuffers/protobuf
 
-Once the application start you can visit `http://localhost:3000/setup` this url will create database tables and admin user once you submit it and redirect to login screen but ignore this page as we got react admin setup in progress.
+* Install the precompiled binary on any OS
 
-### Start react application admin
-    cd react-admin
-    cp .env.example .env    //Basically copy the .env.example file to .env
-    npm i
-    npm start
+https://protobuf.dev/installation/
 
-Visit `localhost:3000/admin` to access the react admin for the portal. Right now react admin does not have many pages redone yet but work in progress.
+* Install Protobuf package on MacOS with `install protobuf`
+
+**Rust**, the systems programing lanugage for backend development.  https://www.rust-lang.org/
+
+**NodeJS**, the frontend web server and frontend toolchain. https://nodejs.org/en
+
+
+### Install Steps
+1. Clone the repository and create the local .env file from the `EXAMPLE.env`.
+
+```bash
+git clone https://github.com/avored/avored-rust-cms.git
+cd avored-rust-cms
+cp EXAMPLE.env .env.dev
+ln -s .env.dev .env
```

Comment:
can you please update this one as 

    cp .env.dev .env 

As idea here was to have 4 different environment. Take a look at this logic. [AvoredConfigProvider.rs](https://github.com/avored/avored-rust-cms/blob/main/src/providers/avored_config_provider.rs#L35)